test/e2e/README.md

```diff
@@ -58,6 +58,7 @@ a specified object store type.  Currently supported cloud provider types are _aw
 E2E tests can be run from the Velero repository root by running `make test-e2e`. While running E2E tests using `make` the E2E test configuration values are passed using `make` variables.
 
 Below is a mapping between `make` variables to E2E configuration flags.
+1. `VELERO_NAMESPACE`: `velero-namespace`. Required.
```

Comment:
I don't see this environment variable referenced outside of this readme in this PR.

I think a better solution would be to update [this line](https://github.com/vmware-tanzu/velero/blob/main/test/e2e/Makefile#L51) in the e2e makefile to be `VELERO_NAMESPACE ?= velero`.

Per the [`make` documentation](https://www.gnu.org/software/make/manual/html_node/Setting.html), `?=` only assigns that value if the variable isn't already set. By setting the default, we don't force users to pass it in if they're only using the default namespace, but do let them change it.

Reply:
Great catch, thank you.